doc/nrf/ug_thread_tools.rst

```diff
@@ -37,15 +37,235 @@ Thread Border Router
-For development purposes, you can use `OpenThread Border Router`_, an open-source Border Router implementation that you can set up either on your PC using Docker or on Raspberry Pi.
-OpenThread Border Router is compatible with Nordic Semiconductor devices.
+* Bidirectional IP connectivity between Thread and Wi-Fi/Ethernet networks.
+* Bidirectional service discovery via mDNS (on Wi-Fi/Ethernet link) and SRP (on Thread network).
+* Thread-over-infrastructure that merges Thread partitions over IP-based links.
+* External Thread Commissioning (for example, a mobile phone) to authenticate and join a Thread device to a Thread network.
+
+
+Configuring OpenThread Border Router
+====================================
+
+There are two ways in which you can set up the Thread Border Router and it depends on your use case.
+
+The recommended option is to build and configure Thread Border Router on RaspberryPi 3B or newer. This option provides
+the most functionality available in OTBR such as Border Routing capabilities (e.g. needed for establishing Thread communication with
+Mobile phone on WiFi network). 
+
+However, for the development, the easiest and quickest way is to run the OTBR Docker container on any Linux-based system.
+This solution is advisable whenever you are only intested in direct communication between your Border Router and the Thread Network, for example,
+to establish IP communication between application running on Linux (e.g. **Python Controller for Matter**) and application running on Thread node.
+
+.. tabs::
+
+   .. tab:: Manual installation
+
+      To setup and configure OpenThread Border Router, please follow the `OpenThread Border Router Codelab`_.
+
+      Note that you may also use nRF52840 Dongle or nRF52840 Development Kit as RCP device, with different transport layer (e.g. UART) instead of presented USB option.
+      In this case, please get familiar with :ref:`ug_thread_tools_tbr_rcp` section below.
+
+   .. tab:: Docker container
+
+      To configure OpenThread Border Router using Docker container, complete the following steps:
+
+      #. Install Docker daemon:
+
+         .. code-block:: console
+            
+            sudo apt update && sudo apt install docker.io
+
+      #. Start the Docker daemon:
+
+         .. code-block:: console
+
+            sudo systemctl start docker
+
+      #. Create an IPv6 network for the OpenThread Border Router container in Docker:
+
+         .. code-block:: console
+
+            sudo docker network create --ipv6 --subnet fd11:db8:1::/64 -o com.docker.network.bridge.name=otbr0 otbr
+
+      #. Download the latest version of the OpenThread Border Router Docker image by running the following command:
+
+         .. code-block:: console
+
+            docker pull openthread/otbr
+
+      #. Start the OpenThread Border Router container using the following command (in the last line, replace ``/dev/ttyACM0``
+         with the device node name of the OpenThread Radio Co-processor):
```

Comment:
Should we link to the code lab or the section below for information on how to build the RCP device? Or even add another step at the beginning?